scst-store/multicluster-setup.hbs.md

```diff
@@ -108,6 +153,66 @@ certificate and authentication token.
 The cluster now has a CA certificate named  `store-ca-cert` and authentication
 token named `store-auth-token` in the namespace `metadata-store-secrets`.
 
+### Copy AMR Cloud Event Handler edit token from the View cluster
+
+Copy the AMR Cloud Event handler token into an environment variable. You use
+this environment variable in the next step.
+
+```bash
+CEH_EDIT_TOKEN=$(kubectl get secrets amr-cloudevent-handler-edit-token -n metadata-store -o jsonpath="{.data.token}" | base64 -d)
+```
+
+## Apply the CA certificate and authentication token to a new Kubernetes cluster
+
+Before you deploy the Build profile, you must apply the CA certificate and
+authentication token from the earlier steps. Then the Build profile deployment
+has access to these values.
+
+To apply the CA certificate and authentication token:
+
+1. With your kubectl targeted at the Build cluster, create a namespace for the CA
+certificate and authentication token.
+
+    ```bash
+    kubectl create ns metadata-store-secrets
+    ```
+
+1. Apply the CA certificate `store_ca.yaml` secret YAML you generated earlier.
+
+    ```bash
+    kubectl apply -f store_ca.yaml
+    ```
+
+1. Create a secret to store the access token. This uses the `AUTH_TOKEN` environment variable.
```

Comment:
Since now you have two auth tokens in this doc. We should change the MDS auth token from `AUTH_TOKEN` to something more specific.